### PR TITLE
(TriggerMesh) adds aws-event-sources chart by TriggerMesh

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

File: packages/aws-event-sources/package.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+url: https://storage.googleapis.com/triggermesh-charts/aws-event-sources-0.1.9.tgz
+packageVersion: 00
```
Comment (PennyScissors) [inline packages/aws-event-sources/package.yaml:2]: Can you change this to `01`?

**Review by PennyScissors — CHANGES_REQUESTED** [review]
Looks good overall, tested deployment in Rancher and it works fine. Just one small change and it should be good to approve and merge. Please make sure you drop the `make charts` commit (34e903a), and create a new one after your changes to pass our validation. The `make charts` commit should always be the last in your PR.

File: packages/aws-event-sources/package.yaml (hunk 1 above)
Comment (sameersbn) [reply packages/aws-event-sources/package.yaml]: updated

**Comment by PennyScissors** [comment]
Hello @sameersbn , could you please drop the last make charts commit, re-run make charts, and create a new commit once again? We recently merged a fix for our scripts that affected existing PRs and charts will need to be regenerated. Thanks!

**Comment by sameersbn** [comment]
> could you please drop the last make charts commit, re-run make charts, and create a new commit once again?

@PennyScissors done!

**Review by PennyScissors — APPROVED** [review]

**Pull request merged**